pkg/discovery/dtofactory/node_entity_dto_builder.go

```diff
@@ -123,6 +123,23 @@ func (builder *nodeEntityDTOBuilder) BuildEntityDTOs(nodes []*api.Node, nodesPod
 			glog.Errorf("Failed to get node properties: %s", err)
 			nodeActive = false
 		}
+		for _, property := range properties{
+			var nodeType string
+			spot := strings.Contains(property.GetName(), util.EKSCapacityType) && property.GetValue() == util.EKSSpot
+			windows := strings.Contains(property.GetName(), util.NodeLabelOS) && property.GetValue() == util.WindowsOS
+			if spot || windows {
+				if spot{
+					nodeType = util.EKSSpot
+				} else if windows {
+					nodeType = util.WindowsOS
+				} else {
+					nodeType = "unknown"
+				}
```

Comment:
I don't understand what "unknown" type means? It is possible that the node is both a spot instance and with windows OS, we just need to print the information out.